### PR TITLE
148/scaffold/Complete VM to responsable relation

### PR DESCRIPTION
**Pull request by MrModder**
See #148

### CODE REVIEW

File: app/api/v_sphere/virtual_machine.rb [hunk 1]
```diff
@@ -271,10 +271,6 @@ def request
       Request.accepted.find { |each| name == each.name }
     end
 
-    def managed_folder_entry
```
Comment (MrModder) [inline app/api/v_sphere/virtual_machine.rb]: Why this change?
It will most likely break inserting VMs into folders!

Comment (regnujAx) [reply app/api/v_sphere/virtual_machine.rb]: Because the method exists twice.

Comment (MrModder) [reply app/api/v_sphere/virtual_machine.rb]: Oh, that makes sense, GitHub just hid that fact from me 🤦‍♂️
